src/clr-angular/data/stack-view/stack-block.ts

```diff
@@ -42,9 +42,9 @@ import { Component, EventEmitter, HostBinding, Input, OnInit, Optional, Output,
   host: { '[class.stack-block]': 'true' },
   animations: [
     trigger('collapse', [
-      state('true', style({ height: 0 })),
-      transition('true => false', [animate('0.2s ease-in-out', style({ height: '*' }))]),
-      transition('false => true', [style({ height: '*' }), animate('0.2s ease-in-out')]),
+      state('true', style({ height: 0, display: 'none' })),
```

Comment:
@mathisscott have we considered using `*ngIf` on the stack children like this:
```
<div @collapse *ngIf="expanded" class="stack-children" >
```
With this, we just need to make tiny changes to this current animation spec to have the same effect while making the solution even more robust.

With current approach, the solution itself is contained in the animation's spec, which makes animation responsible for hiding the nested content as well. So I think we should have a unit test for it. The unit test will help us to figure out why if we break it while trying to retouch the animation effect itself.

Reply:
Oops too late 😬it's merged in. But it's okay. I had a comment only 😄

Reply:
@Shijir 
Yes, this is a good point. Ultimately, we would want (I think) to use the clrIfExpanded directive in stack block. But we could do that without a breaking change.

Given that this fix was worthless for the vSphere team if it introduced a breaking change, I went for the lightest and most non-intrusive change I could manage.